From 25131ff34dc31f0eb562bc4909504f5c8683e1a7 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Fri, 18 Mar 2016 14:55:25 -0600
Subject: [PATCH] move json metadefs to stateless dir

Signed-off-by: Alberto Murillo <alberto.murillo.silva@intel.com>
---
 setup.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 85d9c24..fee7191 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -17,8 +17,8 @@ classifier =
 	Programming Language :: Python :: 2.7
 
 [files]
-data_files = 
-	etc/glance/metadefs = etc/metadefs/*
+data_files =
+        /usr/share/defaults/glance/metadefs = etc/metadefs/*
 packages = 
 	glance
 
-- 
2.5.0

